.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @cloudzero/cirrus
+*       @cloudzero/cirrus
```

Comment:
Weren't we going to add PM here too, or is that in another rep?

Reply:
No need. That is part of the `deployment environment` configuration which is where the signoff is collected. See here: https://github.com/Cloudzero/cloudzero-charts/settings/environments